form.php

```diff
@@ -0,0 +1,145 @@
+<?php
+    $fullName = "";
+    $trimmed = "";
+    $namePattern = '/^[a-zA-Z]+$/';
+    if ($_SERVER['REQUEST_METHOD'] == 'POST' && isset($_POST['Submit'])) {
+        $fName = $_POST['fName'];
+        $lName = $_POST['lName'];
+        $fullName = $fName." ".$lName;
+        $trimmed = preg_replace($namePattern, " ", $fullName);
+    }
+?>
+<?php
+    if(isset($_POST['Submit'])){
+        $fileDestination = "";
+        $file = $_FILES['image'];
+        $fileName = $_FILES['image']['name'];
+        $fileType = $_FILES['image']['type'];
+        $fileError = $_FILES['image']['error'];
+        $fileTmpName = $_FILES['image']['tmp_name'];
+        $fileSize = $_FILES['image']['size'];
+        $allowed = array('jpg','jpeg','png');
+        $fileExt = explode('.', $fileName);
+        $fileActualExt = strtolower(end($fileExt));
+
+        if (in_array($fileActualExt,$allowed)) {
+            if ($fileError === 0) {
+                if ($fileSize>50000000) {
+                    echo "file is too big";
+                }
+                else {
+                    $newFile=uniqid('', true). "." . $fileActualExt;
```

Comment:
Provide space.
TRUE/FALSE should be in caps.

Reply:
Solved.